deployment/windows/jellyfin.nsi

```diff
@@ -0,0 +1,252 @@
+    InstallDir "$APPDATA\Jellyfin"
+  
+;Get installation folder from registry if available
+    InstallDirRegKey HKLM "Software\Jellyfin" "InstallLocation"
+  
+;Request application privileges for service installation
+    RequestExecutionLevel admin
+    CRCCheck on	    
+    !define MUI_ABORTWARNING
+
+;--------------------------------
+;Pages
+
+    !insertmacro MUI_PAGE_LICENSE "$%InstallLocation%\LICENSE"
+    !insertmacro MUI_PAGE_COMPONENTS
+    !define MUI_PAGE_CUSTOMFUNCTION_PRE HideDirectoryPage
+    !insertmacro MUI_PAGE_DIRECTORY	
+    !insertmacro MUI_PAGE_INSTFILES
+	
+    !insertmacro MUI_UNPAGE_CONFIRM
+    !insertmacro MUI_UNPAGE_INSTFILES
+  
+;--------------------------------
+;Languages
+ 
+    !insertmacro MUI_LANGUAGE "English"
+
+;--------------------------------
+;Installer Sections
+
+Section "Install Jellyfin (required)" InstallJellyfin
+; If previous installation is present, run the uninstaller silently
+	StrCmp "$_EXISTINGINSTALLATION_" "YES" RunUninstaller CarryOn
+	
+	RunUninstaller:
+	DetailPrint "Looking for uninstaller at $INSTDIR"
+    FindFirst $0 $1 "$INSTDIR\Uninstall.exe"
+    FindClose $0
+    StrCmp $1 "" CarryOn ; the registry key was there but uninstaller was not found
+
+    DetailPrint "Silently running the uninstaller at $INSTDIR"
+    ExecWait '"$INSTDIR\Uninstall.exe" /S _?=$INSTDIR' $0
+	DetailPrint "Uninstall finished, $0"
+    Delete "$INSTDIR\Uninstall.exe"
+
+	CarryOn:
+
+    SetOutPath "$INSTDIR"
+	
+    SetShellVarContext current ; Local App Data folder for current user only
+    StrCpy $_JELLYFINDATADIR_ "$LOCALAPPDATA\jellyfin\"
+
+; Pack all the files that were just compiled
+; These will be exploded to $INSTDIR
+    File /r $%InstallLocation%\* 
+	
+; Write the installation path into the registry
+    WriteRegStr HKLM "Software\Jellyfin" "InstallLocation" "$INSTDIR"
+
+; Write the uninstall keys for Windows
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\Jellyfin" "DisplayName" "Jellyfin $_JELLYFINVERSION_"
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\Jellyfin" "UninstallString" '"$INSTDIR\Uninstall.exe"'
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\Jellyfin" "DisplayIcon" '"$INSTDIR\Jellyfin.exe",0'
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\Jellyfin" "Publisher" "The Jellyfin project"
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\Jellyfin" "URLInfoAbout" "https://jellyfin.github.io/"
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\Jellyfin" "DisplayVersion" "$_JELLYFINVERSION_"
+    WriteRegDWORD HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\Jellyfin" "NoModify" 1
+    WriteRegDWORD HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\Jellyfin" "NoRepair" 1
+
+;Create uninstaller
+    WriteUninstaller "$INSTDIR\Uninstall.exe"
+
+SectionEnd
+
+Section /o "Jellyfin desktop shortcut" DesktopShortcut
```

Comment:
I'm not sure about needing a desktop shortcut of a service (frankly it might even break with the data access), this should only be possible if people did not install as a service.